### PR TITLE
First pass at DRYing the logger xml.

### PR DESCRIPTION
**Pull request by akash1810**
Using an `include` directive DRY out logger configuration commonality.

Not all apps are created equal, however, so `include` isn't used everywhere. For example, `image-loader` has a root level of `ERROR` whereas other apps are `DEBUG`.

This also only logs AWS ERRORs:

```xml
<logger name="com.amazonaws">
  <level value="ERROR" />
</logger>
```

### CODE REVIEW

File: image-loader/conf/logger.xml [hunk 1]
```diff
@@ -23,6 +23,9 @@
 
     <logger name="play" level="INFO" />
     <logger name="application" level="DEBUG" />
+    <logger name="com.amazonaws">
+        <level value="ERROR" />
+    </logger>
```
Comment (theefer) [inline image-loader/conf/logger.xml]: Why is image-loader different from the others?

File: metadata-editor/conf/logger.xml [hunk 2]
```diff
@@ -2,17 +2,6 @@
...
-        </encoder>
-    </appender>
-
-    <logger name="play" level="INFO" />
-    <logger name="application" level="DEBUG" />
-
-    <root level="DEBUG">
-        <appender-ref ref="CONSOLE"/>
-    </root>
+    <include resource="logger-include.xml" />
```
Comment (theefer) [inline metadata-editor/conf/logger.xml]: Missing the linking file for metadata-editor?

Comment (akash1810) [inline metadata-editor/conf/logger.xml]: Oops! Added.

File: image-loader/conf/logger.xml (hunk 1 above)
Comment (akash1810) [inline image-loader/conf/logger.xml]: Not sure. Have switched the root level to debug in DEV and the logs are not any louder. Will change.

**Comment by kenoir** [comment]
Nice. :+1:

**Comment by akash1810** [comment]
Reverting because TC doesn't support symlinks on server side checkouts because they use JGit on the server - https://youtrack.jetbrains.com/issue/TW-12648.

It looks like [JGit has fixed this](https://bugs.eclipse.org/bugs/show_bug.cgi?id=354367), so maybe a later version of TC will fix this.

**Pull request merged**